railgun/manager/run.go

```diff
@@ -261,6 +269,22 @@ func Run(ctx context.Context, c *config.Config) error {
 		},
 	}
 
+	if ctrlutils.KnativeCRDExist(mgr.GetClient()) == true {
+		setupLog.Info("ingresses.networking.internal.knative.dev crd already deployed. enable knative contorller.")
+		c.KnativeIngressEnabled = util.EnablementStatusEnabled
+		controller := ControllerDef{
+			IsEnabled: &c.KnativeIngressEnabled,
+			Controller: &kongctrl.Knativev1alpha1IngressReconciler{
+				Client:           mgr.GetClient(),
+				Log:              ctrl.Log.WithName("controllers").WithName("Ingress").WithName("KnativeV1Alpha1"),
+				Scheme:           mgr.GetScheme(),
+				Proxy:            prx,
+				IngressClassName: c.IngressClassName,
+			},
+		}
+		controllers = append(controllers, controller)
+	}
```

Comment:
We should add logging that informs the operator when this controller is (automatically) disabled.

Reply:
If user does not create and register controller using manager setup, the controller-runtime/manager does not include the specific operator during running. It is a built-in mechanism.

Reply:
I still think that we should be indicating to the user via logs that the controller was disabled if they tried to specifically enable it, however for the purposes of getting this PR completed I'm OK with skipping on this for now and following up later:

https://github.com/Kong/kubernetes-ingress-controller/issues/1448